models/Service.js

```diff
@@ -26,15 +26,15 @@ Service.init(
             allowNull: false
         },
         cost_basis: {
-            type: DataTypes.ENUM(monthly, yearly),
+            type: DataTypes.ENUM("monthly", "yearly"),
```

Comment:
did you delete the variables that I had declared further up in the file since you decided to just pass in the strings directly? it's fine to do that, I just don't think we should leave the variables there if so

Reply:
I was having issues with the seed. I looked at the documentation and saw that it was passing strings. didn't realize there were variables defined earlier.